examples/kube-state-metrics/kube-state-metrics.yaml

```diff
@@ -318,7 +318,7 @@ spec:
     interval: 30s
     metricRelabeling:
     - action: keep
-      regex: kube_(daemonset|deployment|pod|namespace|node|statefulset|persistentvolume|horizontalpodautoscaler)_.+
```

Comment:
Let's add a YAML comment here, then we don't need to make a second change to update the parent documentation

```
# Curated subset of metrics to reduce costs while populating default set of sample dashboards at
# https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/tree/master/dashboards/kubernetes
# Change this regex to fit your needs for which objects you want to monitor
```

Reply:
added, thanks for the suggestion